docs/modules/jenkins-shared-library/partials/odsComponentStageScanWithTrivy.adoc

```diff
@@ -1,6 +1,6 @@
 // Document generated by render-adoc.go from odsComponentStageScanWithTrivy.adoc.tmpl; DO NOT EDIT.
 
-The "Trivy Security Scan" stage scans the filesystem of the cloned repository using https://github.com/aquasecurity/trivy[Trivy].
+The "Trivy Security Scan" stage scans the filesystem of the cloned repository using https://github.com/aquasecurity/trivy[Trivy] and generates the https://cyclonedx.org/capabilities/sbom[SBOM].
```

Comment:
Why CycloneDX? SPDX has an ISO standard (https://www.iso.org/standard/81870.html) so may have an advantage in terms of regulatory acceptance?

Reply:
To be fair, it was literally because it was mentioned first in Trivy documentation.
@gerardcl also mention that spdx format is the one you use in ods-pipelines and recomended that we set it as default (and I will do so).

Reply:
Unfortunately I do not have extensive real-world experience with both so not sure which one serves our purposes better ... maybe good if the format can be configured?

Reply:
I have made it so most of the parameters can be configured. Only the location where you execute trivy and what to scan (Filesystem) cannot be configured. The rest can be done, with default values

Reply:
I would change it to:
```suggestion
The "Trivy Security Scan" stage scans the filesystem of the cloned repository using https://github.com/aquasecurity/trivy[Trivy] and generates a https://cyclonedx.org/capabilities/sbom[SBOM] report, with https://cyclonedx.org/specification/overview/[CycloneDX] format by default. Check Trivy supported formats https://aquasecurity.github.io/trivy/latest/docs/supply-chain/sbom/[here].
```